src/openforms/variables/tests/factories.py

```diff
@@ -0,0 +1,12 @@
+import factory
+
+from openforms.registrations.contrib.zgw_apis.tests.factories import ServiceFactory
+
+from ..models import ServiceFetchConfiguration
+
+
+class ServiceFetchConfigurationFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = ServiceFetchConfiguration
+
+    service = factory.SubFactory(ServiceFactory)
```

Comment:
can we please stick to a consistent style for models/factories... the `Meta` nested class goes below the field declarations, which conforms to django's model code style on https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#model-style

Reply:
Woops, Black has made me a bit too lazy in formatting :)